server/flaskr/database/postgres/handlers/organization_data_handler.py

```
@@ -0,0 +1,91 @@
+        existing_slugs = cls.get_existing_slugs()
+
+        index = 2
+        while orgSlug in existing_slugs:
+            orgSlug = f"{baseSlug}{index}"
+            index += 1
+
+        try:
+            with get_db_access() as conn:
+                cur = conn.cursor()
+
+                query = f"INSERT INTO organizations (slug, name) values (%s, %s) RETURNING id;"
```

Ruff (0.15.15)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)